.changeset/fuzzy-phones-rule.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/partytown': minor
```

Comment:
```suggestion
'@astrojs/partytown': patch
```

Reply:
Probably should've changed manually, ran the script instead
Sorry for the trouble :)